edgelet/contrib/config/linux/config.yaml

```diff
@@ -16,90 +16,114 @@
-#                       provision with DPS, and only fall back to the provisioning backup
-#                       from a previous successful provisioning if the attempt fails.
-#
-#                       If setting this to false, it's recommended to also set
-#                       dynamic_reprovisioning (see below) to true, so that the device can
-#                       automatically reprovision itself if it needs to.
-#
-# External Settings
+#
+# DPS provisioning with symmetric key attestation
+#     scope_id        - The DPS instance's ID scope
+#     registration_id - The registration ID of this device in DPS.
+#                       For more information regarding DPS registration IDs,
+#                       please see https://docs.microsoft.com/en-us/azure/iot-dps/concepts-device#registration-id
+#     symmetric_key   - The device-specific symmetric key.
+#
+# DPS provisioning with X.509 identity certificate attestation
+#     scope_id        - The DPS instance's ID scope
+#     registration_id - The registration ID of this device in DPS.
+#                       This value is optional. If not specified,
+#                       the common name of the identity certificate will be used
+#                       as the registration ID.
+#                       For more information regarding DPS registration IDs,
+#                       please see https://docs.microsoft.com/en-us/azure/iot-dps/concepts-device#registration-id
+#     identity_cert   - Path of the file containing the Edge device identity certificate.
+#                       The value must be specified as a "file" URI.
+#                       Eg "file:///var/secrets/device-id.pem"
+#     identity_pk     - Path of the file containing the private key of
+#                       the Edge device identity certificate.
+#                       The value must be specified as a "file" URI.
+#                       Eg "file:///var/secrets/device-id.key.pem"
+#
+# External provisioning
 #     endpoint - Required. Value of the endpoint used to retrieve device specific
 #                information such as its IoT hub connection information.
 #
-# Dynamic Re-provisioning Settings
-#     dynamic_reprovisioning - Optional. A flag to opt-in to the dynamic re-provisioning 
-#                              feature. If enabled, IoT Edge on detecting a possible 
-#                              device re-provisioning event will shut down the daemon.
-#                              This is so that on the next daemon startup, the device is 
-#                              set up with the new provisioning information of the device in 
-#                              in IoT Hub.
-#                              For the external provisioning mode specifically, the daemon 
-#                              will notify the external provisioning endpoint about the
-#                              re-provisioning event before shutting down.
+# Miscellaenous settings
```

Comment:
Spelling, should be `Miscellaneous`.

Also should be fixed in the other two files.

Reply:
Fixed.